api/src/Security/Voter/CampIsPrototypeVoter.php

```diff
@@ -17,15 +20,20 @@ class CampIsPrototypeVoter extends Voter {
 
     public function __construct(
         private EntityManagerInterface $em,
+        private ResponseTagger $responseTagger
     ) {}
 
     protected function supports($attribute, $subject): bool {
         return 'CAMP_IS_PROTOTYPE' === $attribute
-        && ($subject instanceof BelongsToCampInterface || $subject instanceof BelongsToContentNodeTreeInterface);
+        && ($subject instanceof BelongsToCampInterface || $subject instanceof BelongsToContentNodeTreeInterface || $subject instanceof Request);
     }
 
     protected function voteOnAttribute(string $attribute, mixed $subject, TokenInterface $token): bool {
-        $camp = $this->getCampFromInterface($subject, $this->em);
+        if ($subject instanceof Request) {
+            $camp = $this->em->getRepository(Camp::class)->find($subject->attributes->get('campId'));
```

Comment:
when do we need the get parameters to vote on the access?

Reply:
https://github.com/api-platform/core/pull/5290 has been merged, so this is now not needed anymore. Reverted in commit https://github.com/ecamp/ecamp3/pull/3610/commits/7703e00507cb6b440cd063e2c9d84946948f21ef